assets/js/babyYoda.js

```diff
@@ -0,0 +1,47 @@
+
+const spawnBabyYoda = (randPosX, randPosY, colour) => {
+    const babyYoda = add([
+      rect(40, 40), // placeholder until we have a sprite
```

Comment:
In the latest push, I moved the sprites to a public folder, so you can add babyYoda's sprite by adding `sprite("grogu"),`. You will then need to remove the `rect(40, 40)` because I think that overlays it. You could also remove the `color(colour)` as that was only to have a colour for the rectangle.